monitoring/README.md

```diff
@@ -0,0 +1,72 @@
+---
+display_name: Monitoring
+description: Monitoring of workspace resources
+maintainer_github: coder
+verified: true
+tags: [monitoring]
+---
+
+# Monitoring
+
+This module adds monitoring of workspace resources.
+
+```tf
+module "monitoring" {
+  source  = "registry.coder.com/modules/monitoring/coder"
+  version = "1.0.0"
```

Comment:
The versioning of modules right now follows the whole repo I believe, so it'd be `v1.0.25`.